src/DefaultHandle.jsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles, withStylesPropTypes } from 'react-with-styles';
+
+import {
+  DEFAULT_HANDLE_WIDTH_UNITS,
+  BACKGROUND_HEIGHT_UNITS,
+  VERTICAL,
+} from './constants/SliderConstants';
+
+import handlePropTypes, { handleDefaultProps } from './propTypes/HandlePropTypes';
+
+export const propTypes = {
+  ...withStylesPropTypes,
+  ...handlePropTypes,
+  'aria-valuetext': PropTypes.string,
+  'aria-label': PropTypes.string,
+};
+
+const defaultProps = {
+  ...handleDefaultProps,
+  'aria-valuetext': undefined,
+  'aria-label': undefined,
+};
+
+function DefaultHandle({
+  css,
+  styles,
+  orientation,
+  disabled,
+  ...passProps
+}) {
+  const {
+    handleRef,
+    theme,
```

Comment:
why not put these in the function argument destructuring?

Reply:
That way we can actually use `passProps` to pass the props to the button rather than `rest`.